envconfig/config.go

```diff
@@ -278,12 +280,14 @@ func AsMap() map[string]EnvVar {
 	return ret
 }
 
-func Values() map[string]string {
-	vals := make(map[string]string)
-	for k, v := range AsMap() {
-		vals[k] = fmt.Sprintf("%v", v.Value)
+func SlogPrint(header string) {
+	m := AsMap()
+	names := maps.Keys(m)
+	slices.Sort(names)
+	slog.Info(header)
+	for _, name := range names {
+		slog.Info(fmt.Sprintf("env: %v=%q", name, m[name]))
```

Comment:
Our goal is to move more towards structured logging to make it easier for analysis/monitoring tools to process the log output from the server.  I think we should try to keep this as a structured log.  I believe the following would make the output a bit more readable in the current text handler, and still retain structure for json handling in the future.

```go
data := []any{}
for k := range names {
	data = append(data, k, m[k])
}
logger.Info(header,
	slog.Group("env", data...))
```

Reply:
But it still in the same line.